### PR TITLE
Webpack update

### PR DESCRIPTION
**Pull request by GabrielLyonB**
## Objetivo
Actualizar _webpacker_ y dependencias.

### CODE REVIEW

File: postcssrc.config.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  plugins: [
+    require('postcss-import'),
+    require('postcss-flexbugs-fixes'),
+    require('postcss-preset-env')({
+      autoprefixer: {
+        flexbox: 'no-2009'
+      },
+      stage: 3
+    })
+  ]
```
Comment (monkeyci) [inline postcssrc.config.js:11]: Missing trailing comma                           comma-dangle

File: postcssrc.config.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  plugins: [
+    require('postcss-import'),
+    require('postcss-flexbugs-fixes'),
+    require('postcss-preset-env')({
+      autoprefixer: {
+        flexbox: 'no-2009'
+      },
+      stage: 3
+    })
```
Comment (monkeyci) [inline postcssrc.config.js:10]: Missing trailing comma                           comma-dangle

File: postcssrc.config.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  plugins: [
+    require('postcss-import'),
+    require('postcss-flexbugs-fixes'),
+    require('postcss-preset-env')({
+      autoprefixer: {
+        flexbox: 'no-2009'
+      },
+      stage: 3
```
Comment (monkeyci) [inline postcssrc.config.js:9]: Missing trailing comma                           comma-dangle

File: postcssrc.config.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  plugins: [
+    require('postcss-import'),
+    require('postcss-flexbugs-fixes'),
+    require('postcss-preset-env')({
+      autoprefixer: {
+        flexbox: 'no-2009'
```
Comment (monkeyci) [inline postcssrc.config.js:7]: Missing trailing comma                           comma-dangle

File: postcssrc.config.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
```
Comment (monkeyci) [inline postcssrc.config.js:1]: Definition for rule 'vue/max-len' was not found  vue/max-len<br>'module' is not defined                          no-undef

File: config/webpack/loaders/vue.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  test: /\.vue(\.erb)?$/,
+  use: [{
+    loader: 'vue-loader',
+  }]
```
Comment (monkeyci) [inline config/webpack/loaders/vue.js:5]: Missing trailing comma                           comma-dangle

File: config/webpack/loaders/vue.js [hunk 7]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (monkeyci) [inline config/webpack/loaders/vue.js:1]: Definition for rule 'vue/max-len' was not found  vue/max-len<br>'module' is not defined                          no-undef

File: config/webpack/environment.js [hunk 8]
```diff
@@ -1,3 +1,19 @@
...
+environment.config.merge({
+  resolve: {
+    alias: {
+      'vue$': 'vue/dist/vue.esm.js',
+    },
+    extensions: ['*', '.js', '.vue', '.json']
+  },
+});
+
+debugger;
```
Comment (monkeyci) [inline config/webpack/environment.js:17]: Using 'DebuggerStatement' is not allowed         no-restricted-syntax

File: config/webpack/environment.js [hunk 9]
```diff
@@ -1,3 +1,19 @@
...
+
+// environment.loaders.delete('nodeModules');
+environment.plugins.prepend('VueLoaderPlugin', new VueLoaderPlugin())
+environment.loaders.prepend('vue', vue)
+environment.config.merge({
+  resolve: {
+    alias: {
+      'vue$': 'vue/dist/vue.esm.js',
+    },
+    extensions: ['*', '.js', '.vue', '.json']
```
Comment (monkeyci) [inline config/webpack/environment.js:13]: Missing trailing comma                           comma-dangle

File: config/webpack/environment.js [hunk 10]
```diff
@@ -1,3 +1,19 @@
 const { environment } = require('@rails/webpacker')
+const { VueLoaderPlugin } = require('vue-loader')
+const vue = require('./loaders/vue')
+
+// environment.loaders.delete('nodeModules');
+environment.plugins.prepend('VueLoaderPlugin', new VueLoaderPlugin())
+environment.loaders.prepend('vue', vue)
```
Comment (monkeyci) [inline config/webpack/environment.js:7]: Missing semicolon                                semi

File: babel.config.js [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+      ],
+      [
+        '@babel/plugin-transform-regenerator',
+        {
+          async: false
+        }
+      ]
+    ].filter(Boolean)
+  }
+}
```
Comment (monkeyci) [inline babel.config.js:72]: Missing semicolon                                semi

File: babel.config.js [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+        }
+      ],
+      [
+        '@babel/plugin-transform-regenerator',
+        {
+          async: false
+        }
+      ]
+    ].filter(Boolean)
+  }
```
Comment (monkeyci) [inline babel.config.js:71]: Missing semicolon                                semi

File: babel.config.js [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+          corejs: false
+        }
+      ],
+      [
+        '@babel/plugin-transform-regenerator',
+        {
+          async: false
+        }
+      ]
+    ].filter(Boolean)
```
Comment (monkeyci) [inline babel.config.js:70]: Missing trailing comma                           comma-dangle

File: babel.config.js [hunk 14]
```diff
@@ -0,0 +1,72 @@
...
+          regenerator: true,
+          corejs: false
+        }
+      ],
+      [
+        '@babel/plugin-transform-regenerator',
+        {
+          async: false
+        }
+      ]
```
Comment (monkeyci) [inline babel.config.js:69]: Missing trailing comma                           comma-dangle

File: babel.config.js [hunk 15]
```diff
@@ -0,0 +1,72 @@
...
+          helpers: false,
+          regenerator: true,
+          corejs: false
+        }
+      ],
+      [
+        '@babel/plugin-transform-regenerator',
+        {
+          async: false
+        }
```
Comment (monkeyci) [inline babel.config.js:68]: Missing trailing comma                           comma-dangle

File: babel.config.js [hunk 16]
```diff
@@ -0,0 +1,72 @@
...
+        {
+          helpers: false,
+          regenerator: true,
+          corejs: false
+        }
+      ],
+      [
+        '@babel/plugin-transform-regenerator',
+        {
+          async: false
```
Comment (monkeyci) [inline babel.config.js:67]: Missing trailing comma                           comma-dangle

File: babel.config.js [hunk 17]
```diff
@@ -0,0 +1,72 @@
...
+          useBuiltIns: true
+        }
+      ],
+      [
+        '@babel/plugin-transform-runtime',
+        {
+          helpers: false,
+          regenerator: true,
+          corejs: false
+        }
```
Comment (monkeyci) [inline babel.config.js:62]: Missing trailing comma                           comma-dangle

File: babel.config.js [hunk 18]
```diff
@@ -0,0 +1,72 @@
...
+        {
+          useBuiltIns: true
+        }
+      ],
+      [
+        '@babel/plugin-transform-runtime',
+        {
+          helpers: false,
+          regenerator: true,
+          corejs: false
```
Comment (monkeyci) [inline babel.config.js:61]: Missing trailing comma                           comma-dangle

File: babel.config.js [hunk 19]
```diff
@@ -0,0 +1,72 @@
...
+        '@babel/plugin-proposal-class-properties',
+        {
+          loose: true
+        }
+      ],
+      [
+        '@babel/plugin-proposal-object-rest-spread',
+        {
+          useBuiltIns: true
+        }
```
Comment (monkeyci) [inline babel.config.js:54]: Missing trailing comma                           comma-dangle

File: babel.config.js [hunk 20]
```diff
@@ -0,0 +1,72 @@
...
+      [
+        '@babel/plugin-proposal-class-properties',
+        {
+          loose: true
+        }
+      ],
+      [
+        '@babel/plugin-proposal-object-rest-spread',
+        {
+          useBuiltIns: true
```
Comment (monkeyci) [inline babel.config.js:53]: Missing trailing comma                           comma-dangle

**Comment by GabrielLyonB** [comment]
Este PR se convirtió en [este](https://github.com/platanus/froggo/pull/232) que funcionó de mejor manera.

**Pull request closed without merging**